handbook/repository/aws-secrets.md

```diff
@@ -0,0 +1,123 @@
+# AWS Secrets
+
+## Prerequisites
+
+- You will need [AWS command line](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed.
+- You will also need access to the AWS account. Ask someone from DevOps to send you an invitation.
+
+## Getting started
+
+1. Configure AWS with SSO, using the following configuration:
+
+```bash
+aws configure sso
+
+SSO start URL [None]: https://island-is.awsapps.com/start
+SSO Region [None]: eu-west-1
+```
+
+Then choose the environnement of your choice. Likely to be `island-is-development01`. You be prompted for the following:
+
+```md
+CLI default client Region [eu-west-1]: <Press Enter>
+CLI default output format [json]: <Press Enter>
+CLI profile name [AWSPowerUserAccess-X]: <Custom name (e.g. dev)> or <Press Enter>
+```
+
+This step will add the new profile to your `~/.aws/config` file.
+
+2. Configure AWS, using the following configuration:
+
+```bash
+aws configure
+
+AWS Access Key ID [None]: <KEY_ID>
+AWS Secret Access Key [None]: <ACCESS_KEY>
+Default region name [eu-west-1]: <Press Enter>
+Default output format [json]: <Press Enter>
+```
+
+You will be able to find these information on the [AWS account](https://island-is.awsapps.com/start).
+
+Then, you will need to open the file created by AWS at `~/.aws/credentials` and add your session token to it.
+
+```bash
+[default]
+aws_access_key_id = <KEY_ID>
+aws_secret_access_key = <ACCESS_KEY>
+aws_session_token = <SESSION_TOKEN>
+```
+
+Voilà, you are now ready to fetch and create secrets.
+
+{% hint style="warning" %}
+Remember, the session token expires every 8 hours so you will need to update it. You can either do `export AWS_SESSION_TOKEN="SESSION_TOKEN"` or change it manually by going into `~/.aws/credentials` and replacing with the new session token generated from your AWS account.
+{% endhint %}
```

Comment:
In my experience, the session token expires every 1 hour, but the SSO session lasts 8 hours. Has this been improved?